definition.go

```diff
@@ -15,7 +16,7 @@ type Definition struct {
 	// Supports template.
 	Command []string `json:"command"`
 	// Replicas control how many instances of Command should run.
-	Replicas int `json:"replicas,omitempty"`
+	Replicas int `json:"replicas"`
```

Comment:
The readme will need to be updated to clarify that this is now a _required_ field in JSON.

An alternate option is to make this a `*int` _with_ `omitempty`, cause then you can differentiate whether it was explicitly configured or not, by checking if the field is `nil`. If it's nil, then you can default it to one, and if it's zero, then you know it was explicitly set to zero.

Reply:
Thank you for the review! It's nice to have support from the Caddy community :)

I changed to use the pointer.